x-pack/plugins/observability_solution/observability_ai_assistant/public/content/starter_prompts.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const defaultStarterPrompts = [
```

Comment:
Placed here so they can be reused in both the assistant_app and the assistant plugin, so they're available in both the conversations app and act as defaults for in the flyout

Reply:
I've added them there.